test/parallel/test-repl-throw-null-or-undefined.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+
+// This test makes sures that the repl does not 
+// crash or emit error when throwing `null|undefined`
+// ie `throw null` or `throw undefined`
+
+const assert = require('assert');
+const repl = require('repl');
+
+const r = repl.start();
+
+r.write('throw null\n');
+r.write('throw undefined\n');
+r.write('.exit\n');
+
+let i = 0;
+r.on('line', function replOutput(output) {
+  const testStatement = i === 0 ? 'null' : 'undefined';
```

Comment:
While this mostly works, the eventemitter manifests pure asynchronous behavior and does not guarentee any order of listener invocation with respect to the order of the associated event, at least by specification. Is it possible to see if `replOutput` function can handle both the possibilities?